tests/testthat/test-apps/reweightView-app/app.R

```diff
@@ -0,0 +1,25 @@
+library(shiny)
+
+ui <- fluidPage(
+  shinyjs::useShinyjs(),
+  riskassessment:::reweightViewUI("reweightInfo")
+)
+
+server <- function(input, output, session) {
+  shinyOptions(golem_options = list(assessment_db_name = "dplyr.sqlite"))
+  
+  user <- reactiveValues(
+    name = "tester",
+    role = "admin"
+  )
+  
+  exportTestValues(
```

Comment:
I really like this pattern you have been following with the module-specific test apps 👍